tests/checkBox.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { expect, test } from '@playwright/test'
+import { beforeEach } from 'node:test'
+
+test.beforeEach(async ({ page }) => {
+	await page.goto('/');
+	await page.locator('div').filter({ hasText: /^Elements$/ }).first().click() //*[text()='Elements']
```

Comment:
can we replace it on page.getByText("Elements")?

Reply:
Yes. I wrote it like this: 
await page.getByText('Elements').click() -the test works